app/javascript/packs/document-capture.tsx

```diff
@@ -143,7 +144,6 @@ const App = composeComponents(
     DocumentCapture,
     {
       onStepChange: extendSession,
-      inPersonFullAddressEntryEnabled: inPersonFullAddressEntryEnabled === 'true',
```

Comment:
This is addressing [Andrew's feedback](https://github.com/18F/identity-idp/pull/8808#discussion_r1268545197) from the prior PR for a cleaner way to pass the feature flag into the Document Capture context